apps/cadence-desktop/cadence-desktop.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Cadence Desktop'
+description: 'Offline CDN http server, organiser and note taker.'
+website: 'https://www.cadence-desktop.com'
+category: ''Developer Tools'
```

Comment:
remove the ' before ```Developer Tools``` and make the line ```category: 'Developer Tools'```
That should fix the CI error

Reply:
Thanks Sanjit, this has now been done.